features/creating_order_with_multiple_items.feature

```diff
@@ -0,0 +1,31 @@
+@managing_orders
```

Comment:
I would also add one scenario that will test removing products from order before placing it. WDYT?